StatPlugin.cs

```diff
@@ -1,15 +1,29 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
+using ServiceStack.Redis;
+
 
 namespace DNWS
 {
   class StatPlugin : IPlugin
   {
+
     protected static Dictionary<String, int> statDictionary = null;
+    protected static RedisManagerPool redisManager = null;
     public StatPlugin()
     {
-      if (statDictionary == null)
+      
+      string redisAttach = Environment.GetEnvironmentVariable("redis"); //redis status
+
+      if (redisAttach != null){ //using redis?
```

Comment:
Hey, what if I set redis=false ?  This is not a good way to check the value of environment!!